data-serving/data-service/src/controllers/preprocessor.ts

```diff
@@ -61,16 +62,42 @@ export const setRevisionMetadata = async (
 ) => {
     const curatorEmail = request.body.curator.email;
 
-    // Find the case if it already exists so we can update its existing
+    // Find the case(s) if it/they already exists so we can update existing
     // metadata.
-    const c = await getCase(request);
+    if (request.body.cases) {
```

Comment:
Might be simpler to have a separate preprocessor, but just a nit/thought.

Reply:
yeah I went back and forth on that, I think you're probably right -- I'll split them